contracts-periphery/test/utils/DeployUmbraTest.sol

```diff
@@ -3,28 +3,64 @@ pragma solidity ^0.8.13;
 
 import "solmate/test/utils/mocks/MockERC20.sol";
 import "test/utils/DSTestPlus.sol";
+import "src/interface/IUmbra.sol";
+import "src/interface/IUmbraHookReceiver.sol";
+import "src/interface/IQuoter.sol";
 
 contract DeployUmbraTest is DSTestPlus {
   address constant umbra = 0xFb2dc580Eed955B528407b4d36FfaFe3da685401;
   MockERC20 token;
-  address constant alice = address(0x202204);
-  address constant bob = address(0x202205);
+  address alice = address(0x202204);
+  address bob = address(0x202205);
   bytes32 constant pkx = "pkx";
   bytes32 constant ciphertext = "ciphertext";
 
-  function setUp() virtual public {
+  function setUp() public virtual {
     deployUmbra();
     createMockERC20AndMint(address(this), 1e7 ether);
     vm.deal(address(this), 1e5 ether);
   }
 
-  function deployUmbra() virtual public {
+  function deployUmbra() public virtual {
     // Deploy Umbra at an arbitrary address, then place the resulting bytecode at the same address as the production deploys.
     vm.etch(umbra, (deployCode("test/utils/Umbra.json", bytes(abi.encode(0, address(this), address(this))))).code);
   }
 
   function createMockERC20AndMint(address addr, uint256 amount) public {
-    token = new MockERC20("Test","TT", 18);
+    token = new MockERC20("Test", "TT", 18);
     token.mint(addr, amount);
   }
-}
\ No newline at end of file
+
+  function _createDigestAndSign(
```

Comment:
This can be internal since it doesn't need to be called by any other contract. I also personally prefer not having the leading underscore for methods in test contracts, since for test contracts only `test*` and `setUp` need to be public and everything else should be internal anyway

```suggestion
  function createDigestAndSign(
```

Reply:
Addressed in https://github.com/ScopeLift/umbra-protocol/pull/341/commits/9c8b8c07c174e352d4fe328137f80b8f4b8a52ec